rules/common.smk

```diff
@@ -12,6 +12,18 @@ def subtypes_by_subtype_wildcard(wildcards):
     }
     return(db[wildcards.subtype])
 
+
+def no_sign_request(_wildcards):
```

Comment:
Is there a benefit to making this a function instead of top-level constants?

```py
NEXTSTRAIN_PUBLIC_BUCKET = "s3://nextstrain-data/"
NO_SIGN_REQUEST = "--no-sign-request" if S3_SRC.startswith(NEXTSTRAIN_PUBLIC_BUCKET) else ""
```

Asking not as a suggestion but out of curiosity as a Snakemake novice.

Reply:
I guess in this case it doesn't really need to be a function. I've just gotten into the habit of using functions because we often depend on the Snakemake wildcard variables in conditionals.

Reply:
I went with the top level constant for `NEXTSTRAIN_PUBLIC_BUCKET` and lambda functions to add the `--no-sign-request` option because we now expect separate S3 URLs for each file since https://github.com/nextstrain/avian-flu/pull/72.